composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "ts/writer",
+    "name": "acsi-tswriter",
```

Comment:
Best is to use the namespace we already use for other modules `acsi/acsi-tswriter`